frontend/src/entities/configuration/configuration-reducer.ts

```diff
@@ -0,0 +1,142 @@
+/**
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import {
+    createAsyncThunk,
+    createSlice,
+    isFulfilled,
+    isPending,
+    PayloadAction,
+} from '@reduxjs/toolkit';
+import axios from '../../shared/util/axios-utils';
+import { defaultConfiguration, IAppConfiguration } from '../../shared/model/app.configuration.model';
+
+const initialState = {
+    loadingAppConfigUpdate: false,
```

Comment:
Name seems a bit misleading - can we rename this `appConfigUpdating` or `appConfigUpdatePending`? There isn't really any `loading` happening with this call.

Reply:
I'm actually just going to remove these altogether as the `formSubmitting` field in the `useValidationReducer` seems to work fine. 

I'll probably need to add a few more formSubmitting fields though to handle all the different loading options (updating the config, importing a config, etc)